scan/main.py

```diff
@@ -22,28 +27,37 @@
 if not openai_api_key:
     raise ValueError("The OPENAI_API_KEY environment variable is not set.")
 
+client = OpenAI(api_key=openai_api_key)
+
+# Define a prompt template for general queries
+general_prompt_template = PromptTemplate(
```

Comment:
Make this upper case to signify it is is a constant

```suggestion
GENERAL_PROMPT_TEMPLATE = PromptTemplate(
```